xml/System.Globalization/PersianCalendar.xml

```diff
@@ -57,8 +57,6 @@
 > [!NOTE]
 >  For information about using the <xref:System.Globalization.PersianCalendar> class and the other calendar classes in the .NET Framework, see [Working with Calendars](/dotnet/standard/datetime/working-with-calendars).  
   
- Dates in the Persian calendar start from the year of the Hijra, which corresponds to 622 C.E. and is the year when Muhammad (PBUH) migrated from Mecca to Medina. For example, the date March 21, 2002 C.E. corresponds to the first day of the month of Farvardeen in the year 1381 Anno Persico.  
```

Comment:
Rather than removing this line, could you please modify it to:

```
- The Persian calendar is a solar Hijri calendar, and starts from the year of the Hijra, which corresponds to 622 C.E. the year when Muhammad (PBUH) migrated from Mecca to Medina.
```

And perhaps add sentence below in the following line:
```
For example, the date March 21, 2002 C.E. corresponds to the first day of the month of Farvardeen in the year 1381 Anno Persico.
```